src/app/components/Image/index.test.tsx

```diff
@@ -284,15 +285,24 @@ describe('Image - AMP pages', () => {
   });
 
   it('should render 2 nested amp-img tags both with alt text', () => {
-    render(<Fixture isAmp />);
+    render(
+      <RequestContextProvider
+        isAmp
+        pageType="article"
+        pathname=""
+        service="afaanoromoo"
+      >
+        <Fixture />
+      </RequestContextProvider>,
```

Comment:
You don't need to add the RequestContextProvider here - it is already included in the render function. 

Instead, you can pass in the value of isAmp as follows:
```suggestion
        <Fixture />, { isAmp }
```